Improvement/artesca 16150 import certificate mutation

JeanMarcMilletScality opened the pull request.

JeanMarcMilletScality committed:
Add hook to import certificate in Zenko CR
Add query to check if extraCACerts exists in Zenko CR as json patch operation is not the same if it exists or not
Update ImportCertificate page to use mutation and query

JeanMarcMilletScality committed:
Update test for ImportCertificate and for mutations

bert-e commented:
# Hello jeanmarcmilletscality,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko-ui/doc/user).


<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @jbwatenbergscality

* @monpote

* @hervedombya

* @damiengillesscality

JeanMarcMilletScality commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @jbwatenbergscality

* @monpote

* @hervedombya

* @damiengillesscality










*The following options are set:* **approve**

JeanMarcMilletScality committed:
improvement on ui:
- remove info message and add space between lines.
- Add certificate to button label
- Put 'or' for Dropzone in lowercase

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbwatenbergscality

* @monpote

* @damiengillesscality

* @chengyanjin

* @hervedombya










*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @hervedombya

* @damiengillesscality

* @jbwatenbergscality

* @monpote










*The following options are set:* **approve**

ChengYanJin reviewed (approved):
LGTM！

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/4.1`



The following branches will **NOT** be impacted:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/3.0`
* `development/3.1`
* `development/3.2`
* `development/4.0`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/4.1`



The following branches have **NOT** changed:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/3.0`
* `development/3.1`
* `development/3.2`
* `development/4.0`



Please check the status of the associated issue ARTESCA-16150.

Goodbye jeanmarcmilletscality.
